src/main/java/com/tradeshift/AppConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.tradeshift;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+import com.tradeshift.service.HelloWorldInterface;
+import com.tradeshift.service.HelloWorldService;
+
+@Configuration
+@ComponentScan({"com.tradeshift"})
```

Comment:
Since you're actually creating the beans in this configuration, there's no need to turn on the component scan. Consider removing this.